hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1855,7 +1874,11 @@
           - team_slug
           - created_at
           - submitted_at
-        filter: {}
+        filter:
+          team:
+            flows:
+              creator_id:
+                _eq: x-hasura-user-id
```

Comment:
Users with the `demoUser` role can only access the records for flows which they created.